src/Pages/Dashboard/FootprintSlideshow/LineGraph.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {
+  VictoryChart,
+  VictoryLabel,
+  VictoryScatter,
+  VictoryLine,
+  VictoryTheme,
+} from "victory";
+
+export default () => {
+  return (
+    <div className="slide">
+      {/* Need to hook up some arithmetic to the numbers being crunched here. */}
+      <div className="lineGraph">
+        <VictoryChart
+          theme={VictoryTheme.material}
+          domain={{ x: [0, 8], y: [0, 8] }}
+        >
+          <VictoryScatter
+            style={{ data: { fill: "#ffffff" } }}
+            data={[
+              { x: 1, y: 3, symbol: "circle", size: 8 },
+              { x: 2, y: 4, symbol: "circle", size: 8 },
+              { x: 3, y: 6, symbol: "circle", size: 8 },
```

Comment:
a bit too repetitive? 
remember, you're in the automation business ;)

Reply:
We are. It is.
This was copied from victory examples site and not used, as we could not in due time make a scatter graph using trees instead of circles. The component is currently unused and will be deleted unless an epiphany or surprise wormhole a couple of weeks back in time occurs.